my_diary/settings.py

```diff
@@ -126,3 +128,20 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
```

Comment:
You also need to set the STATIC_ROOT variable which is also needed in production

Reply:
Thank you,  set the STATIC_ROOT in the subsequent tasks.